.github/workflows/releases.yml

```diff
@@ -0,0 +1,103 @@
+name: Release
+
+on:
+  create:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+env:
+  GO_VERSION: '1.20.6'
+
+permissions:
+  contents: write
+  deployments: write
+
+jobs:
+  # TODO: Any way we can just call build.yml?
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - run: make
+      - run: make test
+      
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        containerd: ["1.6.19", "1.7.0"]
+    env:
+      DOCKER_BUILD_ARGS: "CONTAINERD_VERSION=${{ matrix.containerd }}"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - run: make integration
+
+  generate-artifacts:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup and export variables
+      run: |
+        export release_tag=${GITHUB_REF#refs/*/} # Strip down to raw tag name
+        export release_version=${release_tag/v/} # Remove v from tag name
+        echo "release_version=$release_version" >> $GITHUB_ENV
```

Comment:
nit: is `release_version` still needed as a GitHub environment variable? Usage appears to be only scoped to this step.

Reply:
Good catch, forgot to remove it from the previous action I tried using